manifests/init.pp

```diff
@@ -73,6 +75,7 @@
   Boolean $generate = true,
   Boolean $regenerate = false,
   Boolean $deploy = true,
+  String[1] $ca_key_password = $certs::params::ca_key_password,
```

Comment:
What's the reason for moving this parameter to another line?

I also like the data type, but should we enforce at least X characters? for NSS we use 10 and I think that's a good number.

Reply:
This wasn't a parameter before. The other line in the diff was where it was defined inside the class. I also increased the string size to 10 per your suggestion.